README.md

```diff
@@ -1,30 +1,68 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app).
+<!-- Create a readme with:
+title
+description
+all technologies used
+1 or 2 screenshots
+setup instructions
+deployment instructions -->
 
-## Getting Started
+# Bist du Teppich online-shop
 
-First, run the development server:
+## Description
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+The project at hand is an online shop of interior decoration items created with Next.js.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+The shop features following pages and functionality:
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+- a products page which also contains a search bar
+- a page for each single product
+- a shopping cart page containing a list where products appear when you click on the "Add to cart" button on the single product page, which also shows the total price of all products
+- a checkout page which shows the total and asks for shipping and payment information (when you click on "Go to Checkout" on the shopping cart page it goes here)
+- a thank you page after a checkout has been completed
+- the header shows a shopping cart with the current number of items on all pages.
 
-## Learn More
+Products page: <img src="/public/screenshot1.png" width="500">
+Cart: <img src="/public/screenshot2.png" width="500">
 
-To learn more about Next.js, take a look at the following resources:
+## Technologies used
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+Bist du Teppich is a Next.js app which makes use of PostgresQL database. Migrations are set up with Ley. It uses cookies to keep track of items in cart and their amount. Some pages and components are written using Typescript. The project contains testing with Jest and Cypress. Deployment was carried out with Heroku.
 
-You can check out [the Next.js GitHub repository](https://github.com/zeit/next.js/) - your feedback and contributions are welcome!
+## Setup instructions
 
-## Deploy on Vercel
+### Database Setup
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/import?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+Copy the .env.example file to .env and add the database connection information.
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+You'll also need PostgreSQL for this.
+
+PostgreSQL Installation instructions
+
+Follow the instructions from the PostgreSQL step on https://www.postgresql.org/docs/10/runtime.html
```

Comment:
I guess this is the wrong link? You meant to link to the [UpLeveled system setup](https://github.com/upleveled/system-setup)?